DnsCachePlugin/main.c

```diff
@@ -230,6 +275,15 @@ VOID ShowStatusMenu(
                     }
                 }
                 break;
+            case 2:
+                HGLOBAL hMem = GlobalAlloc(GMEM_MOVEABLE, cacheEntryName->Length * 2 + 2);
+                memcpy(GlobalLock(hMem), cacheEntryName->Buffer, cacheEntryName->Length*2);
+                GlobalUnlock(hMem);
+                OpenClipboard(0);
+                EmptyClipboard();
+                SetClipboardData(CF_UNICODETEXT, hMem);
+                CloseClipboard();
+                break;
```

Comment:
PH has a helper function called PhSetClipboardString and you can replace this entire block of code with just this:
PhSetClipboardString(ListViewHandle, &cacheEntryName->sr);